python/cudf/cudf/tests/test_groupby.py

```diff
@@ -1532,3 +1538,26 @@ def test_groupby_nonempty_no_keys(pdf):
         lambda: gdf.groupby([]),
         compare_error_message=False,
     )
+
+
+@pytest.mark.parametrize(
+    "by,data",
+    [
+        # ([], []),  # error?
```

Comment:
libcudf raises here, but maybe we should early-return anyway. Looking into this.

Reply:
Same issue as https://github.com/rapidsai/cudf/issues/7728. Likely should handle in a separate fix.

Reply:
Should we add a catch in the Python side temporarily to at least raise a nice `NotImplementedError`?

Reply:
Fixed. Another instance of us catching the libcudf error string..